Kudu.Services/Deployment/PushDeploymentController.cs

```diff
@@ -216,6 +369,16 @@ private async Task<HttpResponseMessage> PushDeployAsync(ZipDeploymentInfo deploy
                     deploymentInfo.RepositoryUrl = zipFilePath;
                 }
             }
+            else if (deploymentInfo.Deployer == Constants.OneDeploy)
```

Comment:
// Copy the request body to a temp file.
// It will be moved to the appropriate directory by the Fetch handler